### DISCUSSION TITLE
Ghelper start as minimized

### DISCUSSION THREAD

**Discussion by distresscode** [seerge/g-helper#2789] [General]
I have Ghelper "run on startup" turned on. It's working fine, but on each restart, the app is opened and I have to hit the X to close it. Is it possible to have it start as minimized in the taskbar?

**Comment by seerge** [comment]
@distresscode hello, it already starts as minimized if runs from a windows Task Scheduler (on startup). If you run it manually by clicking on icon / exe - then it will show main window.

If something is wrong with a startup task of the app - try to uncheck and then check ``Run on startup`` again

**Reply by distresscode** [reply]
I'm not sure about the windows task scheduler method, but yes, I set it up using the `run on startup` toggle. I'll report back if rechecking doesn't fix it.

**Reply by distresscode** [reply]
Re-checking the box didn't fix it. Here's a screenshot of the task scheduler. 

![image](https://github.com/seerge/g-helper/assets/2686608/fca0d20d-533b-4103-86c5-4c660ff4292b)
![image](https://github.com/seerge/g-helper/assets/2686608/4d698691-afc8-40ee-bceb-55485fa41245)
![image](https://github.com/seerge/g-helper/assets/2686608/b869e64f-0a46-4bd2-b191-eff479484f3e)

**Reply by distresscode** [reply]
BTW, when re-checking the `run on startup` option, UAC pops up asking for permission. Is that normal?

**Reply by distresscode** [reply]
I unchecked `run on startup` and Ghelper isn't listed in the task scheduler anymore. Yet, the app is launching and open after restarting the PC.

**Reply by distresscode** [reply]
I'll delete the app and redo the set up later today and report back if the issue persists.

**Reply by seerge** [reply]
@distresscode if app is still launching even w/o task - it means something else is running this :) So try to figure out what is that (did you set it to run on startup it somehow manually ?)

**Reply by distresscode** [reply]
You're right! It's still toggled on in Apps> Startup. 

![image](https://github.com/seerge/g-helper/assets/2686608/8c954b05-608f-4fd8-89b4-8afacf86e10c)

**Reply by seerge** [reply]
@distresscode G-Helper is not able to add itself there (it can only add task to Task Scheduler). Did you use some other tool (or windows itself) to add it here ?

Just remove it from here, that's probably a reason it popup up as a window ...

**Reply by distresscode** [reply]
I must have done it myself when trying to set it up last week. TBH I had forgotten what I did which is why it didn't occur to me. Anyhow, it's working as intended now. Thank you very much for the help!

**Comment by distresscode** [comment]
solved - thanks again, Serge!
